src/zjs_performance.c

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016, Linaro Limited.
+#ifdef BUILD_MODULE_PERFORMANCE
+
+// ZJS includes
+#include "zjs_util.h"
+
+static jerry_value_t zjs_performance_now(const jerry_value_t function_obj,
+                                         const jerry_value_t this,
+                                         const jerry_value_t argv[],
+                                         const jerry_length_t argc)
+{
+    if (argc != 0)
```

Comment:
I'm not sure what you mean by "javascript allows" here. WebIDL seems to be a strictly typed representation and it doesn't specify that now accepts arbitrary parameters: https://www.w3.org/TR/hr-time/#the-performance-interface . I'd personally consider it an error to pass parameters to function which doesn't accept them. But then I consider following an error:

~~~
console.log(obj.misspelled_property)
~~~

and JavaScript doesn't. So, I'll be happy to follow if, with the above in mind, you'll confirm that the check should be removed.